numba/core/funcdesc.py

```diff
@@ -84,12 +84,15 @@ def lookup_module(self):
         """
         Return the module in which this function is supposed to exist.
         This may be a dummy module if the function was dynamically
-        generated.
+        generated. Raise exception if the module can't be found.
         """
         if self.modname == _dynamic_modname:
             return _dynamic_module
         else:
-            return sys.modules[self.modname]
+            try:
+                return sys.modules[self.modname]
+            except:
+                raise ModuleNotFoundError(f"can't compile {self.qualname}: import of module {self.modname} failed")
```

Comment:
Please could this line be wrapped to 80 chars? Thanks.

Reply:
updated in 576a4b8

Reply:
Thanks!